test/functional/admin/consultations_controller_test.rb

```diff
@@ -131,6 +131,16 @@ class Admin::ConsultationsControllerTest < ActionController::TestCase
     end
   end
 
+  view_test "create should show 'Processing' tag if all variants are not uploaded" do
```

Comment:
Can this please be renamed to "if not all variants are uploaded"/"some variants are missing" -> the implication in "all variants are not uploaded" is that every single one must be missing for the processing tag to show.